### ISSUE TITLE
Issue with serverless template parsing

### ISSUE THREAD

**Issue by stell-aelsabbahy** [stelligent/cfn-model#34]
```
13:54:14 + scl enable rh-ruby23 'gem install cfn-nag'
13:54:15 Successfully installed cfn-nag-0.3.53
13:54:15 Parsing documentation for cfn-nag-0.3.53
13:54:15 Done installing documentation for cfn-nag after 0 seconds
13:54:15 1 gem installed
13:54:15 + find CloudFormation -name '*.template'
13:54:15 + xargs -t -n 1 cfn_nag
13:54:15 cfn_nag CloudFormation/Serverless.template 
13:54:15 /var/lib/jenkins/.gem/ruby/gems/cfn-model-0.1.24/lib/cfn-model/transforms/serverless.rb:29:in `object_key_from_uri': undefined method `join' for nil:NilClass (NoMethodError)
13:54:15 Did you mean?  JSON
13:54:15    from /var/lib/jenkins/.gem/ruby/gems/cfn-model-0.1.24/lib/cfn-model/transforms/serverless.rb:39:in `replace_serverless_function'
13:54:15    from /var/lib/jenkins/.gem/ruby/gems/cfn-model-0.1.24/lib/cfn-model/transforms/serverless.rb:11:in `block in perform_transform'
13:54:15    from /var/lib/jenkins/.gem/ruby/gems/cfn-model-0.1.24/lib/cfn-model/transforms/serverless.rb:9:in `each'
13:54:15    from /var/lib/jenkins/.gem/ruby/gems/cfn-model-0.1.24/lib/cfn-model/transforms/serverless.rb:9:in `perform_transform'
13:54:15    from /var/lib/jenkins/.gem/ruby/gems/cfn-model-0.1.24/lib/cfn-model/parser/transform_registry.rb:20:in `perform_transforms'
13:54:15    from /var/lib/jenkins/.gem/ruby/gems/cfn-model-0.1.24/lib/cfn-model/parser/cfn_parser.rb:41:in `parse'
13:54:15    from /var/lib/jenkins/.gem/ruby/gems/cfn-nag-0.3.53/lib/cfn-nag/cfn_nag.rb:82:in `audit'
13:54:15    from /var/lib/jenkins/.gem/ruby/gems/cfn-nag-0.3.53/bin/cfn_nag:70:in `<top (required)>'
13:54:15    from /var/lib/jenkins/bin/cfn_nag:23:in `load'
13:54:15    from /var/lib/jenkins/bin/cfn_nag:23:in `<main>'
13:54:15 Build step 'Execute shell' marked build as failure
```

template file:
```
AWSTemplateFormatVersion: '2010-09-09'
Transform: 'AWS::Serverless-2016-10-31'
Description: 'Create Serverless application.'
Parameters:
  Environment:
    Description: Name of the AWS deployment environment
    Type: String
    AllowedValues:
    - rd
    - di
    - qa
    - ct
    - pr
  ProductIdentifier:
    Description: Tag for the name of the product
    Type: String
Resources:
  ServerlessFunc:
    Type: 'AWS::Serverless::Function'
    Properties:
      FunctionName: !Sub '${Environment}-${ProductIdentifier}'
      Handler: handler.hello
      Runtime: python3.6
      CodeUri: ../app
      Description: >-
        Sample application1
      MemorySize: 128
      Timeout: 3
      Role: !Sub 'arn:aws:iam::${AWS::AccountId}:role/${Environment}-${ProductIdentifier}-Role'
      Environment:
        Variables:
          AWS_ENV: !Ref Environment
```

**Comment by jeffb4** [comment]
This should now be fixed (problematic template added as a test case). Please close if so, or ping me if not, thanks!

**Comment by jeffb4** [comment]
Confirmed resolved in "0.1.25", closing. Thanks for opening the issue Ahmed :)
